DEBUG    temporalio.worker._activity:_activity.py:276 Running activity wait_cancel (token b'\n$f032a326-ed6d-485e-81c8-68533bd3a6fc\x12-workflow-26588603-c4ad-4e53-a18d-2e633b03682a\x1a$249863f5-8786-4dcb-b8da-2c52b623c5dc \x05(\x012\x011B\x0bwait_cancelJ\x08\x08\x01\x10\xa8\xa9@\x18\x01')
DEBUG    temporalio.activity:_activity.py:440 Starting activity ({'activity_id': '1', 'activity_type': 'wait_cancel', 'attempt': 1, 'namespace': 'default', 'task_queue': '45e8825b-9350-4681-a1f8-3ea7f1ab3d59', 'workflow_id': 'workflow-26588603-c4ad-4e53-a18d-2e633b03682a', 'workflow_run_id': '249863f5-8786-4dcb-b8da-2c52b623c5dc', 'workflow_type': 'ActivityTimeoutWorkflow'})
DEBUG    temporalio.activity:_activity.py:255 Recording heartbeat with details () ({'activity_id': '1', 'activity_type': 'wait_cancel', 'attempt': 1, 'namespace': 'default', 'task_queue': '45e8825b-9350-4681-a1f8-3ea7f1ab3d59', 'workflow_id': 'workflow-26588603-c4ad-4e53-a18d-2e633b03682a', 'workflow_run_id': '249863f5-8786-4dcb-b8da-2c52b623c5dc', 'workflow_type': 'ActivityTimeoutWorkflow'})
DEBUG    temporalio.activity:_activity.py:255 Recording heartbeat with details () ({'activity_id': '1', 'activity_type': 'wait_cancel', 'attempt': 1, 'namespace': 'default', 'task_queue': '45e8825b-9350-4681-a1f8-3ea7f1ab3d59', 'workflow_id': 'workflow-26588603-c4ad-4e53-a18d-2e633b03682a', 'workflow_run_id': '249863f5-8786-4dcb-b8da-2c52b623c5dc', 'workflow_type': 'ActivityTimeoutWorkflow'})
DEBUG    temporalio.activity:_activity.py:255 Recording heartbeat with details () ({'activity_id': '1', 'activity_type': 'wait_cancel', 'attempt': 1, 'namespace': 'default', 'task_queue': '45e8825b-9350-4681-a1f8-3ea7f1ab3d59', 'workflow_id': 'workflow-26588603-c4ad-4e53-a18d-2e633b03682a', 'workflow_run_id': '249863f5-8786-4dcb-b8da-2c52b623c5dc', 'workflow_type': 'ActivityTimeoutWorkflow'})
DEBUG    temporalio.worker._workflow_instance:_workflow_instance.py:1788 Workflow raised failure with run ID 249863f5-8786-4dcb-b8da-2c52b623c5dc
temporalio.exceptions.TimeoutError: activity StartToClose timeout

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/runner/work/sdk-python/sdk-python/temporalio/worker/_workflow_instance.py", line 1767, in _run_top_level_workflow_function
    await coro
  File "/home/runner/work/sdk-python/sdk-python/temporalio/worker/_workflow_instance.py", line 831, in run_workflow
    result = await self._inbound.execute_workflow(input)
  File "/home/runner/work/sdk-python/sdk-python/temporalio/testing/_workflow.py", line 493, in execute_workflow
    return await super().execute_workflow(input)
  File "/home/runner/work/sdk-python/sdk-python/temporalio/worker/_interceptor.py", line 328, in execute_workflow
    return await self.next.execute_workflow(input)
  File "/home/runner/work/sdk-python/sdk-python/temporalio/worker/_workflow_instance.py", line 2099, in execute_workflow
    return await input.run_fn(*args)
  File "/home/runner/work/sdk-python/sdk-python/tests/worker/test_workflow.py", line 1447, in run
    await workflow.execute_activity(
  File "/home/runner/work/sdk-python/sdk-python/temporalio/workflow.py", line 2011, in execute_activity
    return await _Runtime.current().workflow_start_activity(
  File "/home/runner/work/sdk-python/sdk-python/temporalio/worker/_workflow_instance.py", line 1423, in run_activity
    return await asyncio.shield(handle._result_fut)
temporalio.exceptions.ActivityError: Activity task timed out
INFO     temporalio.worker._worker:_worker.py:523 Beginning worker shutdown, will wait 0:00:00 before cancelling activities
DEBUG    temporalio.worker._activity:_activity.py:216 Cancelling activity b'\n$f032a326-ed6d-485e-81c8-68533bd3a6fc\x12-workflow-26588603-c4ad-4e53-a18d-2e633b03682a\x1a$249863f5-8786-4dcb-b8da-2c52b623c5dc \x05(\x012\x011B\x0bwait_cancelJ\x08\x08\x01\x10\xa8\xa9@\x18\x01', reason: 3
DEBUG    temporalio.worker._activity:_activity.py:527 Completing activity with completion: task_token: "\n$f032a326-ed6d-485e-81c8-68533bd3a6fc\022-workflow-26588603-c4ad-4e53-a18d-2e633b03682a\032$249863f5-8786-4dcb-b8da-2c52b623c5dc \005(\0012\0011B\013wait_cancelJ\010\010\001\020\250\251@\030\001"
result {
  completed {
    result {
      metadata {
        key: "encoding"
        value: "json/plain"
      }
      data: "\"Got cancelled error, cancelled? True\""
    }
  }
}